src/l500/l500-serializable.cpp

```diff
@@ -0,0 +1,55 @@
+//// License: Apache 2.0. See LICENSE file in root directory.
+//// Copyright(c) 2018 Intel Corporation. All Rights Reserved.
+
+#include "l500-serializable.h"
+#include "../../../third-party/json.hpp"
+
+namespace librealsense
+{
+    using json = nlohmann::json;
+
+    l500_serializable_base::l500_serializable_base(std::shared_ptr<hw_monitor> hw_monitor, synthetic_sensor & depth_sensor)
```

Comment:
Why there is a need for L500_serizable and how is it different for D400_serializable ?
There is no SKU-specific stuff.